lib/workers/repository/process/lookup/index.ts

```diff
@@ -327,6 +327,34 @@ export async function lookupUpdates(
 
         res.updates.push(update);
       }
+    } else if (
+      is.string(currentValue) &&
+      config.replacementName &&
+      config.replacementVersion
+    ) {
+      config = mergeConfigConstraints(config);
+      const dependency = clone(await getPkgReleases(config));
+      if (!dependency) {
+        const warning: ValidationMessage = {
+          topic: depName,
+          message: `Failed to look up dependency ${depName}`,
+        };
+        logger.debug({ dependency: depName, packageFile }, warning.message);
+        // TODO: return warnings in own field
+        res.warnings.push(warning);
```

Comment:
are you sure this works as expected? please test on real repo

Reply:
I just extended the repo and yes for me it is working as expected, throwing the warning if the dependency that is currently used (and is going to be replaced) is not found.

Log output showing that the same result is produced for valid currentValue and (the new) invalid currentValues:
```
 {
   "depName": "tkulmburgA",
   "currentValue": "myversion",
   "replaceString": "tkulmburgA:myversion",
   "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
   "datasource": "docker",
   "depType": "stage",
   "depIndex": 2,
   "updates": [],
   "warnings": [
     {
       "topic": "tkulmburgA",
       "message": "Failed to look up docker dependency tkulmburgA"
     }
   ],
   "versioning": "docker"
 },
 {
   "depName": "tkulmburgB",
   "currentValue": "1.1",
   "replaceString": "tkulmburgB:1.1",
   "autoReplaceStringTemplate": "{{depName}}{{#if newValue}}:{{newValue}}{{/if}}{{#if newDigest}}@{{newDigest}}{{/if}}",
   "datasource": "docker",
   "depType": "final",
   "depIndex": 3,
   "updates": [],
   "warnings": [
     {
       "topic": "tkulmburgB",
       "message": "Failed to look up docker dependency tkulmburgB"
     }
   ],
   "versioning": "docker"
 }
```